gravity-forms/gw-disable-submit-button-until-required-fields-are-filled-out.php

```diff
@@ -14,15 +14,23 @@ class GW_Disable_Submit {
 
 	public static $script_output = false;
 
+	public $form_id;	
+
 	public function __construct( $form_id ) {
 
-		add_action( "gform_pre_render_{$form_id}", array( $this, 'maybe_output_script' ) );
-		add_action( "gform_register_init_scripts_{$form_id}", array( $this, 'add_init_script' ) );
+		$this->form_id = $form_id;
+
+		add_action( 'gform_pre_render', array( $this, 'maybe_output_script' ) );
+		add_action( 'gform_register_init_scripts', array( $this, 'add_init_script' ) );
 
 	}
 
 	public function maybe_output_script( $form ) {
 
+		if ( $form['id'] !== $this->form_id ) {
```

Comment:
```suggestion
		if ( $form['id'] != $this->form_id ) {
```

Typically, in snippets we generally stray away from eqeqeq so they're a bit more forgiving in terms of configuration.